docs/reference/alpha-web-ui.md

```diff
@@ -45,11 +45,14 @@ Start with bootstrapping a React app with `create-react-app`
 npx create-react-app your-feast-ui
 ```
 
-Then, in your app folder, install Feast UI and its peer dependencies. Assuming you use yarn
+Then, in your app folder, install Feast UI and optionally its peer dependencies. Assuming you use yarn
 
 ```
 yarn add @feast-dev/feast-ui
-yarn add @elastic/eui @elastic/datemath @emotion/react moment prop-types inter-ui react-query react-router-dom use-query-params zod typescript query-string d3 @types/d3
```

Comment:
Why remove all of these?

Reply:
These are no longer in the peer dependencies, so there's no need to install them anymore, at least not for Feast UI to work.

Reply:
So these are now considered internal dependencies of Feast UI, mostly irrelevant for apps using it as a module.